src/cli/index.ts

```diff
@@ -90,8 +91,37 @@ $$$$$$\\   $$$$$$$\\  $$\\  $$$$$$\\   $$$$$$$ |$$\\  $$\\  $$\\  $$$$$$\\  $$$$
     .option("--dry-run", "dry run (skip actually publishing)")
     .option("-d, --debug", "show debug logs")
     .option("--ci", "Continuous Integration mode")
+    .option(
+      "-n, --name [name]",
+      "Name of the pre-built or released contract (such as nft-drop)",
+    )
+    .option(
+      "--cv, --contract-version [version]",
```

Comment:
I'm guessing we can't use --version here?

Reply:
yea its reserved